### ISSUE TITLE
Arrow functions produces SyntaxError after upgrading to 1.0.12

### ISSUE THREAD

**Issue by tellnes** [Sage/streamlinejs#311]
```js
foo((_) => {
  bar.baz()
})
```

```
./test._js:9
    bar.baz();
       ^

SyntaxError: Unexpected token .
    at exports.runInThisContext (vm.js:54:16)
    at Module._compile (module.js:375:25)
    at Module._compile (/node_modules/streamline/lib/register.js:52:12)
    at Object.._js (/node_modules/streamline/lib/register.js:65:10)
    at runScript (/node_modules/streamline/lib/command.js:127:25)
    at Object.exports.run (/node_modules/streamline/lib/command.js:149:3)
    at Object.<anonymous> (/node_modules/streamline/bin/_node:11:18)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
    at Module.load (module.js:345:32)
```

**Comment by bjouhier** [comment]
@tellnes I just published 1.0.13 which fixes the problem. I also fixed a problem with `this` which was incorrectly bound in arrow functions.

**Comment by tellnes** [comment]
Thanks for fast response :)
